### PR TITLE
Refactor RGBD

### PR DESCRIPTION
**Pull request by markus-hinsche**
joined work with @prajurock 

**Motivation**: The RGBD code does not yet adhere to our coding standards. 

**Solution**: This PR improves readabilitity. 

**Changes:**
- Sort imports, Use global variable for height and weight, remove commented code
- Clean docstrings
- Make global variables uppercase, polish import
- Rename to use snake case, Fix import order
- Use does_path_belong_to_codes() function to remove code duplication
- Make small adjustments to `fusion.py` and `calibration.py`
- Refactor `rgbd.py` and `get_timestamp.py`

# How Has This Been Tested?

We ran the command `python rgbd.py --input $CGM_ML_DIR/data/anon-bmz-version5.0_incomplete/qrcode/ --output $CGM_ML_DIR/data/anon-bmz-version5.0_incomplete_converted/qrcode/`

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests
- [x] New and existing unit tests pass locally with my changes
- [ ] My Code is review by 1 People

### CODE REVIEW

File: src/common/rgbd_toolkit/cgm_fusion/normals_with_open3d.py [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
-enable_visualization = False
-downsample = True
+ENABLE_VISUALIZATION = False
+DOWNSAMPLE = True
 
 if __name__ == "__main__":
 
     print("Load a ply point cloud, print it, and render it")
-    pcd = o3d.io.read_point_cloud("/data/home/cpfitzner/test.pcd")
+    pcd = open3d.io.read_point_cloud("/data/home/cpfitzner/test.pcd")
```
Comment (gangooteli) [inline src/common/rgbd_toolkit/cgm_fusion/normals_with_open3d.py:12]: Please read the path using ArgumentParser

File: src/common/rgbd_toolkit/cgm_fusion/normals_with_open3d.py [hunk 2]
```diff
@@ -44,14 +44,14 @@
     print("")
 
     print("Load a polygon volume and use it to crop the original point cloud")
-    vol = o3d.visualization.read_selection_polygon_volume(
+    vol = open3d.visualization.read_selection_polygon_volume(
```
Comment (gangooteli) [inline src/common/rgbd_toolkit/cgm_fusion/normals_with_open3d.py:47]: Please read the path using ArgumentParser

**Review by gangooteli — CHANGES_REQUESTED** [review]

**Comment by markus-hinsche** [comment]
@gangooteli I agree that argparse would be nicer but this was not introduced by this PR. We can further refactor `normals_with_open3d.py` but we were not sure if it is used at all, so maybe it can even be deleted. 

Prajwal will take a closer look in a follow-up PR: https://dev.azure.com/cgmorg/ChildGrowthMonitor/_workitems/edit/418

File: src/common/rgbd_toolkit/cgm_fusion/normals_with_open3d.py (hunk 1 above)
Comment (gangooteli) [reply src/common/rgbd_toolkit/cgm_fusion/normals_with_open3d.py:12]: Will be handled in new PR https://dev.azure.com/cgmorg/ChildGrowthMonitor/_workitems/edit/418

File: src/common/rgbd_toolkit/cgm_fusion/normals_with_open3d.py (hunk 2 above)
Comment (gangooteli) [reply src/common/rgbd_toolkit/cgm_fusion/normals_with_open3d.py:47]: Will be handled in new PR https://dev.azure.com/cgmorg/ChildGrowthMonitor/_workitems/edit/418

**Review by gangooteli — APPROVED** [review]
LGTM

**Pull request merged**